src/InvestJob.js

```diff
@@ -7,6 +7,9 @@ const logger = getLogger(__filename.split('/').slice(-1)[0]);
 
 const HASH_ARRAY_KEY = 'workspaceHashes';
 const MAX_CACHED_JOBS = 30;
+const investJobStore = localforage.createInstance({
+  name: "InvestJobs"
+});
```

Comment:
That's a good question. The documentation on it is really sparse: https://localforage.github.io/localForage/#multiple-instances-createinstance

My take is that there is `name` and `storename`. `storename` will make a new `store` in the db where as `name` will make a completely separate db. I went for the cautious completely separate db approach. 

Since there is no `loadinstance` or any other kind of handling I believe behind the scenes that if a db by `name` does exist `createinstance` simply loads that db. Since we are currently hard-coding and managing the two db names, I don't think it's really necessary to do any checking that they already exist or not, unless we want the user to be able to delete them permanently. Then maybe we would want a check for existence. 

There's a lot that we could set up using the `localforage.config` as well. But so far our use cases are pretty basic that I went with simplest implementation.

Reply:
> Since there is no `loadinstance` or any other kind of handling I believe behind the scenes that if a db by `name` does exist `createinstance` simply loads that db.

Cool, this is the part I was concerned about. Also found this thread that comes to the same conclusion https://github.com/localForage/localForage/issues/719

Does our mock of `createInstance` operate in the same way? That might be important for testing loading values from a pre-existing db.

Reply:
Great question Dave. The `createInstance` mock is just a one time `Class` creator that replicates `localforage` functions to operate on an object. Yea, I guess that same implementation could be useful in a test where you render the component and then tear it down and re-render it to see if the values have been loaded.... But I almost think I'd rather be able to mock different defaults into `createInstance` instead... At the moment it might be better to not have `__mocks__/localforage.js` and just do the mocking in the test files.

Reply:
@dcdenu4 I do think the `InvestJob` constructor function is a model of how this could all work without any more mocking.

Reply:
In essence it provides an interface to the localforage db that is accessible outside the context of the react layer. So it can be used in the app's components and it can be used just the same in test setup.